ostree-ng.sh

```diff
@@ -485,13 +489,129 @@ greenprint "📥 Downloading the image"
 sudo composer-cli compose image "${COMPOSE_ID}" > /dev/null
 ISO_FILENAME="${COMPOSE_ID}-${INSTALLER_FILENAME}"
 modksiso "${ISO_FILENAME}" "/var/lib/libvirt/images/${ISO_FILENAME}"
-sudo rm -f "${ISO_FILENAME}"
```

Comment:
Please do not delete ISO_FILENAME, you will use and mount it.

Reply:
Ok, thanks! It has been moved to line 513